scipy/optimize/tests/test_linprog.py

```diff
@@ -1312,6 +1312,24 @@ def test_bug_10124(self):
                       method=self.method, options=self.options)
         _assert_success(res, desired_x=[10, -3], desired_fun=-22)
 
+    def test_bug_10349(self):
+        """
+        Test for tolerance issue on LPs with equality constraints
+        https://github.com/scipy/scipy/issues/10349
+        """
+        A_eq = np.array([[1, 1, 0, 0, 0, 0],
+                         [0, 0, 1, 1, 0, 0],
+                         [0, 0, 0, 0, 1, 1],
+                         [1, 0, 1, 0, 0, 0],
+                         [0, 0, 0, 1, 1, 0],
+                         [0, 1, 0, 0, 0, 1]])
+        b_eq = np.array([221, 210, 10, 141, 198, 102])
+        c = np.concatenate((0, 1, np.zeros(4)), axis=None)
+        with suppress_warnings() as sup:
+            sup.filter(OptimizeWarning, "A_eq does not appear...")
+            res = linprog(c, A_eq=A_eq, b_eq=b_eq)
```

Comment:
```suggestion
            res = linprog(c, A_ub, b_ub, A_eq, b_eq, bounds,
                          method=self.method, options=self.options)
```
Notice that linprog is called that way in almost every test. It's not super important that it be called with `A_ub`, `b_ub`, and `bounds` as inputs (which are all set to `None`) above (lines 226-230), but it is important to call `linprog` with `method=self.method, options=self.options` so that the test is run with all the different combinations of solver and options. (Notice the subclasses of `LinprogCommonTests` where `self.method` and `self.options` are set.)
Anyway, it's easier to write tests if you just copy the entirety of these two lines as they appear elsewhere. Keeps things consistent and helps avoid mistakes.